docs/products/opensearch/howto/sample-dataset.rst

```diff
@@ -0,0 +1,169 @@
+Sample dataset
+--------------
+Here you will find different sample datasets for OpenSearch and how to interact with them.
+
+Before exploring the available datasets, we need an OpenSearch server to load the data on. A quick and easy way to get a server is through `Aiven for OpenSearch <https://aiven.io>`_. If don't have an Aiven account yet, `sign up <https://console.aiven.io/signup?utm_source=github&amp;utm_medium=organic&amp;utm_campaign=devportal&amp;utm_content=repo>`_ and enjoy our free trial!
```

Comment:
Now that these instructions have moved to devportal, this could instead become "We will need an OpenSearch service to use for this example" and then the rest of the instructions could follow I think. It would also be OK to include the signup link.